kubernetes/k8ssandra-values.yaml

```diff
@@ -0,0 +1,38 @@
+cassandra:
+  version: "4.0.0"
```

Comment:
4.0.1?

Reply:
I'm going to pull the Kubernetes config from this PR, it is still a WIP that I included inadvertently.